projects/ngrx.io/content/guide/migration/v15.md

```diff
@@ -221,3 +221,26 @@ export class TestComponent {
 }
 ```
 
+## Deprecations
```

Comment:
I don't think we need to add this here.
We usually keep this for migrations between v14 -> v15.
This could also be confusing when peoples are on v15.0.0, in which this method is not available (I don't know why they shouldn't upgrade to the latest v15, but it happens 🤷).

I like that it's already documented though, maybe we should mention the required NgRx version, or already add a v16 file?

Reply:
I agree with Tim. Maybe we need a separate guide for deprecations between each major

Reply:
What do you think about adding a note in the title, like this:

```diff
## Deprecations

### @ngrx/router-store

-#### Renamed `getSelectors` Function
+#### Renamed `getSelectors` Function (Introduced in v15.2)

The `getSelectors` function is deprecated in favor of `getRouterSelectors`.
```

Reply:
Angular does the following in [upgrade.angular.com](https://update.angular.io/?v=15.0-16.0)
I like the additional note
> Warning: Plans for releases after the current major release are not finalized and may change. These recommendations are based on scheduled deprecations.

They also version their website with the `rc` and `next` tag

![image](https://user-images.githubusercontent.com/28659384/211373444-49cd89aa-737f-41ce-998d-fd34e6fcce2e.png)

I don't think we need `rc`, but we could add a `next` migration page with the above note where all the deprecations can be added.

When we release that version, we can transform the deprecations to breaking change.


I think doing this would make our intentions clear.

If that's not good or would cause confusion, I also like Marko's proposal :)

Reply:
Currently, our `next.ngrx.io` page points at what's latest on `master`, so we may have some extra work there. I think Marko's note on the deprecations page is a good start